config/example/config.toml

```diff
@@ -116,7 +120,7 @@ admin_rank_colour_map = [
 	{ name = "Community Manager", colour = "#e91e63" },
 	{ name = "Game Admin", colour = "#238afa" },
 	{ name = "Trial Admin", colour = "#7fb6fc" },
-	{ name = "PR Reviewer", colour = "#c27c0e" },
+	{ name = "Developer", colour = "#c27c0e" },
```

Comment:
Is it worth migrating people with +VIEWRUNTIMES to this role?

Reply:
We sometimes give viewruntimes to people not on the dev team, although it is rare